PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -46,15 +46,53 @@ static bool isAllDigits(const generic_string &str)
 	return std::all_of(str.begin(), str.end(), ::isdigit);

```

Comment:
Ah I see what you mean. Is there some other form of `isdigit()` that would work? `find_first_not_of()` should work but a native function to check the character would probably be a bit more efficient and clearer...in the end shouldn't really matter though.